docs/parity-status.md

```diff
@@ -1,77 +1,31 @@
+All [React Native Components and APIs](https://reactnative.dev/docs/components-and-apis) that are a part of the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort are now all supported in React Native for Windows.
 
-If there is a discrepancy here, or features are not being tracked that should be, please [submit an issue under vnext](https://github.com/microsoft/react-native-windows/issues/new?labels=vnext&template=vnext.md) to let us know.
+For a more closely monitored look at our work in progress, check out the:
+- [API Completion](https://github.com/microsoft/react-native-windows/labels/API%20Completion) label on issues
+- [GitHub project board - Core Features](https://github.com/microsoft/react-native-windows/projects/7)
+- [GitHub project board - Community Module Support](https://github.com/microsoft/react-native-windows/projects/23)
 
-You can keep track of our component and module API parity more closely on our [GitHub project borad](https://github.com/microsoft/react-native-windows/projects/7).
+> Please feel free to leave comments on give us a 👍 on issues to help us prioritize better!
 
-### Components
-Below are a list of components and their status towards parity between the lean core components you'd find on iOS and Android and the functionality we have today.
+A few methods or props may be missing on some types and we are actively working to add support for those in our [upcoming milestones](https://github.com/microsoft/react-native-windows/milestones).
 
-|Component| `vnext` version Status | `vnext` Issues remaining | `current` version Status |
-|:-|:-|:-|:-|
-|ActivityIndicator|**Complete**|-|**Complete**|
-|Button|**Complete**|-|**Complete**|
-|FlatList|**Complete**|-|**Complete**|
-|Image|Partial|[Image project board](https://github.com/microsoft/react-native-windows/projects/18)|Partial|
-|ScrollView|Partial|[ScrollView project board](https://github.com/microsoft/react-native-windows/projects/17)|Partial|
-|Switch|**Complete**|-|**Complete**|
-|Text|**Complete** *(Some properties not supported due to device platform differences, docs coming soon)*|**Complete**|**Complete**|
-|TextInput|Partial|[TextInput project board](https://github.com/microsoft/react-native-windows/projects/20)|Partial|
-|KeyboardAvoidingView|**Complete**|-|Not supported|
-|View|Partial (some Style props left)|[View Style props project board](https://github.com/microsoft/react-native-windows/projects/19)|**Complete**|
-|VirtualizedList|Partial|No Issues Logged|**Complete**|
+If you encounter a supported API that is not being tracked, or an unsupported API that should be added, please [submit an issue](https://github.com/microsoft/react-native-windows/issues/new/choose) to let us know.
 
-### Modules
-Below are a list of modules and their status towards parity between the lean core components you'd find on iOS and Android and the functionality we have today.
+## Supported Community Modules

```

Comment:
Repasting a blurb that should be added to the end here. I think you should point to the Community modules project board here and the issue template for adding support for community modules is also different - so worth calling these out separately from API parity above.

In addition, we are working on adding Windows support to popular and highly requested community modules. You can keep track on the progress of both these efforts through the [Community Modules support project board](https://github.com/microsoft/react-native-windows/projects/23).  
If there is an unsupported community module that you don’t see in our roadmap, please [submit a support request](https://github.com/microsoft/react-native-windows/issues/new?assignees=&labels=Extensions&template=react-native-windows--vnext--module-request.md&title=Support+react-native-foo) and let us know.

Reply:
Should I remove the community module project board link from the top of the document then and just keep it under this section?

Reply:
The reason is - there are 2 aspects to the community modules stuff. Lean core refactored and just general popular modules. I want to make sure people understand we are contributing on both aspects and know to ask us for requests on both.